### PR TITLE
Changes the minimum amount of officers needed to enable unlimited assistants

### PR DESCRIPTION
**Pull request by Landerlow**
Currently, when there are more than five officers on the station, an unlimited amount of assistants can join, even if proper jobs are open. This change increases the minimum officer amount necessary for unlimited assistants to spawn from 5 to 8, in the hopes that when needed, people will play more regular jobs. 

🆑 
edit: Changes the minimum amount of officers to allow unlimited assistants from 5 to 8
/ 🆑

### CODE REVIEW

**Comment by adamkad1** [comment]
8 is a lot of sec scrubs

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -205,7 +205,7 @@
 			var/datum/job/hos = job_master.GetJob("Head of Security")
 			count += (officer.current_positions + warden.current_positions + hos.current_positions)
 			if(job.current_positions > (config.assistantratio * count))
-				if(count >= 5) // if theres more than 5 security on the station just let assistants join regardless, they should be able to handle the tide
+				if(count >= 8) // if theres more than 5 security on the station just let assistants join regardless, they should be able to handle the tide
```
Comment (SamHPurp) [inline code/modules/mob/new_player/new_player.dm]: You didn't even change the comment.

**Comment by SamHPurp** [comment]
Leading merge commits.... RIP

File: code/modules/mob/new_player/new_player.dm (hunk 1 above)
Comment (Landerlow) [inline code/modules/mob/new_player/new_player.dm]: Indeed. Oops?

**Comment by Landerlow** [comment]
Honestly this merge commits thing WHY does that keep happening?

**Comment by Landerlow** [comment]
I always use a clean base for any PR's, yet this is the second time this is happening, I have no idea why.

**Comment by SamHPurp** [comment]
You aren't updating your master branch fully before creating your branch.

**Comment by Landerlow** [comment]
Yes I am ...

**Comment by Landerlow** [comment]
Re-installing time I think. RIP.

**Comment by SamHPurp** [comment]
... Clearly not.

Are you using Github for Desktop? Their sync and update functionality is shit. Use the Gitshell option in Github for Desktop and run these commands on your master branch.

`git fetch ParadiseSS13`
`git merge ParadiseSS13/master`
`git push`

**Comment by alexkar598** [comment]
Using --ff-only on the merge command will ensure you will not create merge commits and if it needs to do one it will instead abort

**Pull request closed without merging**